### ISSUE TITLE
Prevent Nesting

### ISSUE THREAD

**Issue by renatoi** [brigade/scss-lint#246]
It was previously mentioned on PR #143 that nesting is good and not nesting is bad. I disagree with that and I believe it's exactly the opposite, as nesting could be a hindrance for readability. There should be an option to prevent nesting.

The only option we have in the linter so far is `force_nesting` for `MergeableSelector`, however there's no way to prevent nesting.

**Bad**

``` scss
h1 {
  color: #fff;
  &.new {
    color: #000;
  }
}
```

**Good**

``` scss
h1 {
  color: #fff;
}
h1.new {
   color: #000;
}
```
